### DISCUSSION TITLE
HDMI screen does not turn off after updating to 20231210

### DISCUSSION THREAD

**Discussion by akovan** [rern/rAudio#1529] [Bug Reports]
Hi,
Immediately after reboot, screen off timer works fine, but if player was used and then stopped/paused, screen no longer turns off.

**Comment by akovan** [comment]
Changed this line:
[https://github.com/rern/rAudio/blob/25519cbfcd627eca4572cb2099a67d0de8df1d0a/srv/http/bash/status-push.sh#L40](https://github.com/rern/rAudio/blob/25519cbfcd627eca4572cb2099a67d0de8df1d0a/srv/http/bash/status-push.sh#L40)

to
`        [[ $state == play ]] && sudo xset -dpms || sudo xset +dpms`

and it works again.

**Reply by rern** [reply]
Thank you for this. A new update should be released shortly.
